scripts/GetClrDbg.sh

```diff
@@ -95,8 +134,17 @@ fi
 
 # For the rest of this script we can assume the working directory is the install path
 
+echo 'Info: Determinig Runtime ID'
+__RuntimeID=
+get_dotnet_runtime_id
+if [ -z $__RuntimeID ]; then
+    echo "Error: Unable to determine dotnet Runtime ID"
```

Comment:
Thinking about it more: debugging probably doesn't work in 2035, so I would suggest using 1.0.0-beta-002173